blocks/jsonpath.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": ["jsonpath"],
```

Comment:
You don't use a npm package, you should include the utils folder in the `includes` instead.